go.mod

```diff
@@ -8,7 +8,9 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/mattermost/mattermost-plugin-api v0.0.12-0.20200917103517-788aef584648
 	github.com/mattermost/mattermost-server/v5 v5.28.0
+	github.com/microcosm-cc/bluemonday v1.0.15
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.6.1
+	golang.org/x/net v0.0.0-20210726213435-c6fcb2dbf985 // indirect
```

Comment:
Nit: Remove indirect dependency

Reply:
@hanzei tried removing it , while deploying it creates problem

Reply:
Using `go1.16.7` I can remove this line and the plugin correctly compiles. What is the error that you are seeing?

Reply:
@hanzei  upgraded to 1.16 it works now